backend/src/main/webapp/blackswan/script.js

```diff
@@ -0,0 +1,47 @@
+// Copyright 2020 Google LLC
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     https://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+const FLAG_LOCAL = 1;
+/** 
+ * When testing locally, need to append host that python service 
+ * runs on to requests.
+ */
+const LOCAL = FLAG_LOCAL === 1 ? 'http://localhost:8889' : '';
```

Comment:
will this be always 8889 (as defined by app engine or some other config), or is it possible that can it be some other port?

Reply:
It is based on the port defined in the `python` service. It would be be great if this can be obtained from a shared config between the `java` service and `python` service. 

For now, I will set the port number as a constant in the same file if thats okay. Once I have the shared config set up (hopefully in a future PR), I can get the port number from the shared config.